src/components/ModalWindow/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styles from './modal-window.scss';
+import settings from 'settings';
+import CloseButton from 'components/CloseButton';
+
+const Modal = React.createClass({
```

Comment:
Perhaps, but we are triggering `this.props.hideModalWindow();` so I thought it was good here.

Likewise we could just `dispatch(hideModalWindow())` in the container too and we wouldn't need the dispatch action in this component.